panic: reflect: call of reflect.Value.FieldByName on int64 Value on (*Error).GetError

```
panic: reflect: call of reflect.Value.FieldByName on int64 Value

goroutine 82 [running]:
reflect.flag.mustBe(...)
	/usr/local/go/src/reflect/value.go:233
reflect.Value.FieldByName({0x1163900?, 0xc000968308?, 0xc000968308?}, {0x135ec36?, 0x7f418cbc05b8?})
	/usr/local/go/src/reflect/value.go:1355 +0x1c8
grest.dev/grest.(*Error).GetError(0x0?, {0x1163900?, 0xc000968308})
	/.../grest/error.go:50 +0xfb
github.com/my-user/my-app/middleware.(*logHandler).send(_, {{0xc00004a840, 0x20}, {0x135d444, 0x2}, {{0xc000968000, 0x3}, {0xc000968130, 0x10}, {0x0, ...}, ...}, ...}, ...)
	/home/my-user/my-app/middleware/log.go:75 +0xd2
created by github.com/my-user/my-app/middleware.(*logHandler).New in goroutine 32
	/home/my-user/my-app/middleware/log.go:35 +0x279
exit status 2
```

--- a/error.go
+++ b/error.go
@@ -47,8 +47,10 @@ func (e *Error) GetError(err any) *Error {
 		if rv.Kind() == reflect.Pointer {
 			rv = rv.Elem()
 		}
-		if rvCode := rv.FieldByName("Code"); rvCode.Kind() == reflect.Int {
-			code = int(rvCode.Int())
+		if rv.Kind() == reflect.Struct {
+			if rvCode := rv.FieldByName("Code"); rvCode.Kind() == reflect.Int {
+				code = int(rvCode.Int())
+			}
 		}
 		return NewError(code, er.Error())
 	}

